crates/quick_action_bar/src/quick_action_bar.rs

```diff
@@ -121,15 +121,22 @@ impl Render for QuickActionBar {
                 }
             },
         );
+        let mut assistant_icon = true;
+        if let Some(elements) = WorkspaceSettings::get_global(cx)
```

Comment:
It would be better to leave this expressed in the current structure using `.when`.